pkg/registry/core/service/allocator/interfaces.go

```diff
@@ -44,6 +44,6 @@ type Snapshottable interface {
 	Restore(string, []byte) error
 }
 
-type AllocatorFactory func(max int, rangeSpec string) (Interface, error)
+type AllocatorFactory func(max int, rangeSpec string, offset int) (Interface, error)
```

Comment:
why not using `AllocatorWithOffsetFactory`?

Reply:
do you really need to modify this?

Reply:
well, I didn't notice it. `AllocatorWithOffsetFactory` is better, I correct it now.